### PR TITLE
GIT.03.Hosting

### PR DESCRIPTION
**Pull request by usertiger**
Everything is done, connected with ssh - successfully

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (pluhin) [inline .vscode/settings.json:1]: Please remove this, it is from your VSC

or add line(s) in gitignore

for this

**Review by pluhin — COMMENTED** [review]
This "demo/.DS_Store" same case

File: .vscode/settings.json (hunk 1 above)
Comment (usertiger) [reply .vscode/settings.json]: removed

**Comment by usertiger** [comment]
> This "demo/.DS_Store" same case

removed

**Review by pluhin — CHANGES_REQUESTED** [review]
One more
`demo/.DS_Store `
need to remove

**Comment by usertiger** [comment]
> One more `demo/.DS_Store ` need to remove

sorry, done

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**